docs/4-user-guide/2-zarf-packages/1-zarf-packages.md

```diff
@@ -4,72 +4,69 @@ sidebar_position: 1
 
 # Understanding Zarf Packages
 
-A Zarf package is a single tarball archive that contains everything you need to deploy a system or capability while fully disconnected. Zarf packages are defined by a `zarf.yaml` file.
+Zarf offers a comprehensive solution for deploying system software or capabilities while fully disconnected. This is accomplished through the use of a single tarball archive, which includes all necessary components and is defined by a `zarf.yaml` file.
 
-Zarf packages are built while 'online' and connected to whatever is hosting the dependencies your package definition defined. When being built, all these defined dependencies are downloaded and stored within the archive. Because all the dependencies are now within the tarball, the package can be deployed to disconnected systems that don't have a connection to the outside world.
+Zarf Packages are built while connected to the internet and to whatever is hosting the dependencies your package definition defined. This allows for the retrieval and storage of defined dependencies within the archive. As a result, the package can be deployed on disconnected systems without requiring a connection to the outside world.
```

Comment:
```suggestion
Zarf Packages are created while you are connected to internet or intranet resources defined in a Zarf package configuration.  This allows these resources to be pulled and stored within a package archive and be deployed on disconnected systems without requiring a connection to the outside world.
```

I may be getting tired, but I feel this is a bit better wording.

Reply:
"internet or intranet resources defined in a Zarf package configuration." This portion didn't make too much sense to me. Maybe "internet or intranet and resources are defined in a Zarf Package configuration."

Reply:
`the internet or an intranet and resources are defined in a Zarf Package configuration` could make sense but in the way I had phrased it internet or intranet are being used as adjectives for "resources" i.e. internet resources or intranet resources.